api/v1alpha1/release_types.go

```diff
@@ -130,6 +130,9 @@ type ProcessingInfo struct {
 
 // ValidationInfo defines the observed state of the release validation.
 type ValidationInfo struct {
+	// Automated indicates whether the Release was created by the integration service
```

Comment:
I don't really like to mention services in the comments in case the name changes or those services get split (e.g. has and build services). What about:
```suggestion
	// Automated indicates whether the Release was created as part of an automated process or manually by an end-user
```